### PR TITLE
liquibase-3.5.3 (update) adding PostgreSQL support

### PR DESCRIPTION
**Pull request by nyarly**

### CODE REVIEW

File: pkgs/development/tools/database/liquibase/default.nix [hunk 1]
```diff
@@ -1,12 +1,14 @@
 { stdenv, fetchurl, writeText, jre, makeWrapper, fetchMavenArtifact
-, mysqlSupport ? true, mysql_jdbc ? null }:
+, mysqlSupport ? true, mysql_jdbc ? null
+, postgresSupport ? true, postgresql_jdbc ? null }:
```
Comment (Profpatsch) [inline pkgs/development/tools/database/liquibase/default.nix]: Would rename to `postgresqlSupport` to keep the same naming.

**Review by Profpatsch — CHANGES_REQUESTED** [review]
Have you tested calling postgres from a liquibase specification?

It should be possible to create an automated test for postgresql, see https://gitlab.techcultivation.org/sangha/sangha-deployment/blob/master/containers/helpers/postgres-template.nix#L31 for an example on how to run postgres inside the build.
If you don’t want to invest the time, a manual test would be enough as well for now.

**Comment by nyarly** [comment]
@Profpatsch Do you have a liquibase changelog to work from? I've been trying to get this working for my own purposes (and if I can, I'll report it as manually tested), but as a result I don't have anything I can actually test *with*.

**Comment by Profpatsch** [comment]
> Do you have a liquibase changelog to work from?

No, sorry. I’m not sure I even did use liquibase back then. :)

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

**Comment by nyarly** [comment]
This has been manually tested now. I'd like to puzzle out how to do the automated testing... time limits forbid at the moment.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -11974,7 +11974,9 @@ with pkgs;
     postgresql96
     postgresql100;
 
-  postgresql_jdbc = callPackage ../servers/sql/postgresql/jdbc { };
+    postgresql_jdbc = callPackage ../servers/sql/postgresql/jdbc {
+      fetchMaven = callPackage ../development/java-modules/m2install.nix { };
+    };
```
Comment (Profpatsch) [inline pkgs/top-level/all-packages.nix]: nitpick: should be dedented by two spaces.

**Review by Profpatsch — APPROVED** [review]

**Comment by aanderse** [comment]
@nyarly are you able to continue with this PR?

**Comment by Ekleog** [comment]
(triage) Closing for now due to lack of activity, @nyarly feel free to react if you want to continue working on this!

**Pull request closed without merging**